.prettierrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "https://json.schemastore.org/prettierrc",
+  "singleQuote": true,
```

Comment:
I think we want double quote here, it seems to change *slightly* fewer files. We somehow managed to have even number of single and double quotes usage.

Reply:
While I do personally tend to prefer single quotes I don't mind changing this if we really want to, but I think doing so because of the number of files changed isn't the best reason since this is a "do once" operation and you can exclude commits in `git blame` and other places using a setting I'll dig up in a bit

Reply:
The reason I prefer double quotes is it matches go strings being double quotes so it keeps things slightly more consistent.

Reply:
Now _that_ is a good reason 😄